pricegraph/src/api/price_estimation.rs

```diff
@@ -33,7 +33,7 @@ impl Pricegraph {
             orderbook.fill_optimal_transitive_order_if(inverse_pair, |flow| {
                 last_exchange_rate = Some(flow.exchange_rate);
                 false
-            })?;
+            });
```

Comment:
**NOTE**: This is a fix to a regression introduced in the `Pricegraph` API during refactoring because of insufficient test coverage. I am working on a separate PR that addresses this issue that should be merged before this one.